src/main/java/org/opentestsystem/ap/ims/rest/VersionedItemApi.java

```diff
@@ -0,0 +1,54 @@
+package org.opentestsystem.ap.ims.rest;
+
+import lombok.extern.slf4j.Slf4j;
+import org.opentestsystem.ap.common.model.Item;
+import org.opentestsystem.ap.ims.model.ItemResource;
+import org.opentestsystem.ap.ims.service.VersionedItemService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.CacheControl;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequestMapping("/api/v1/versioned-items")
+public class VersionedItemApi {
+    private final VersionedItemService versionedItemService;
+
+    @Autowired
```

Comment:
The Autowired annotation is not required anymore when doing constructor injection.  I forget when that came along in Spring, maybe 4.x.  Not big deal to put it but if you don't want to anymore you don't have to.

Reply:
Thanks, but I am kind "old fashioned" when it comes to this magic and prefer to use it. Unless you have a strong objection?

Reply:
No objections.  It is fine to have it there and makes it clear.